include/boost/json/detail/value_conversion.hpp

```diff
@@ -0,0 +1,173 @@
+//
+// Copyright (c) 2020 Krystian Stasiowski (sdkrystian@gmail.com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/vinniefalco/json
+//
+
+#ifndef BOOST_JSON_DETAIL_VALUE_CONVERSION_HPP
```

Comment:
Please rename this file to `value_traits.hpp` (since it consists mostly of metafunctions)